src/main/kotlin/com/bugsnag/android/gradle/BugsnagManifestUuidTask.kt

```diff
@@ -1,24 +1,17 @@
 package com.bugsnag.android.gradle
 
-import com.android.build.gradle.api.BaseVariant
-import com.android.build.gradle.api.BaseVariantOutput
-import com.android.build.gradle.tasks.ManifestProcessorTask
 import com.bugsnag.android.gradle.internal.property
 import org.gradle.api.DefaultTask
-import org.gradle.api.Project
-import org.gradle.api.file.Directory
-import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.file.RegularFileProperty
-import org.gradle.api.logging.Logger
 import org.gradle.api.model.ObjectFactory
 import org.gradle.api.provider.Property
-import org.gradle.api.provider.Provider
 import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.TaskAction
-import java.io.File
-import java.nio.file.Paths
 import javax.inject.Inject
 
 abstract class BaseBugsnagManifestUuidTask(objects: ObjectFactory) : DefaultTask() {
```

Comment:
Is the separate base class still needed?

Reply:
Nope. I'll remove it as well.